ss-client/build.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/sh
 
-if [ $# -ne 4 ]; then
-    echo "Usage: ./build.sh KEY-FILE CERTIFICATE SERVER-ADDRESS FILENAME"
-    echo "E.g. ./build.sh ss.key cert.pem doggoat.de Updater"
+if [ $# -ne 3 ]; then
+    echo "Usage: ./build.sh CERTIFICATE SERVER-ADDRESS FILENAME"
+    echo "E.g. ./build.sh cert.pem doggoat.de Updater"
```

Comment:
Somewhere else you changed targetHost to include port. build.sh usage example is not updated to account for this change.

```suggestion
    echo "E.g. ./build.sh cert.pem doggoat.de:48879 Updater"
```